Source/WebCore/platform/graphics/FontPlatformData.h

```diff
@@ -166,6 +166,7 @@ class FontPlatformData {
 
     static FontPlatformData cloneWithSize(const FontPlatformData&, float);
     void updateSizeWithFontSizeAdjust(const FontSizeAdjust&);
+    static float usedFontSize(float, const FontSizeAdjust&, float);
```

Comment:
That's a good idea, will do, thanks

Reply:
However, the name `adjusted` might be confusing because this is just the adjusted size regarding @font-face size-adjust, but not regarding font-size-adjust. I'm thinking to call it `adjustedSizeForFontFace` although sounds a bit silly.